Lib/fontParts/base/font.py

```diff
@@ -1537,42 +2100,72 @@ def _isCompatible(self, other, reporter):
     # mapping
     # -------
 
-    def getReverseComponentMapping(self):
-        """
-        Get a reversed map of component references in the font.
-        {
-        'A' : ['Aacute', 'Aring']
-        'acute' : ['Aacute']
-        'ring' : ['Aring']
-        etc.
-        }
+    def getReverseComponentMapping(self) -> ReverseComponentMappingType:
+        """Get a reversed map of all component references in the font.
+
+        This method creates a :class:`dict` mapping each component glyph
+        name in the font to a :class:`set` containing the composite
+        glyph names that include the comoponent.
+
+        Note that one glyph can have multiple unicode values, and a
+        unicode value can have multiple glyphs pointing to it.
+
+        :return: A :class:`dict` of component glyph names mapped to sets
+            of composite glyph names.
+
+        Example::
+
+            {
+            'A': {'Aacute', 'Aring'},
```

Comment:
should be `()` not `{}`

Reply:
Ghanged, I've also made the example more complete:

```python
"""
...

Example::

    >>> mapping = font.getReverseComponentMapping()
    >>> mapping
    {'A': ('Aacute', 'Aring'), 'acute': ('Aacute',),
    'ring': ('Aring',)  , ...}

"""
```

Reply:
THANK YOU!